src/main/scala/com/github/unchama/seichiassist/menus/stickmenu/SecondPage.scala

```diff
@@ -477,6 +478,36 @@ object SecondPage extends Menu {
       )
     }
 
+    val ItemNameRemoveButton:Button = {
+      val iconItemStack = new IconItemStackBuilder(Material.DIAMOND_BARDING, durability = 1)
+        .title(s"$YELLOW$UNDERLINE${BOLD}名義削除システム")
+        .lore(List(
+          s"$RESET${GREEN}所有者表記を取り除きたいアイテムの所有者を",
+          s"$RESET${GOLD}「所有者：なし」RESET${GREEN}に変更します",
+          s"$RESET${GREEN}出てきたインベントリに",
+          s"$RESET${GREEN}名義を削除したいアイテムを入れて",
+          s"$RESET${GREEN}escキーを押してください",
+          s"$RESET${DARK_GRAY}たまにアイテムが消失しますが",
+          s"$RESET${DARK_GRAY}補償はしていません(ごめんなさい)",
+          s"$RESET${DARK_GRAY}神に祈りながら交換しよう",
+          s"$RESET$DARK_RED${UNDERLINE}クリックで開く"
+        ))
```

Comment:
```suggestion
        .lore(
          s"$RESET${GREEN}所有者表記を取り除きたいアイテムの所有者を",
          s"$RESET${GOLD}「所有者：なし」RESET${GREEN}に変更します",
          s"$RESET${GREEN}出てきたインベントリに",
          s"$RESET${GREEN}名義を削除したいアイテムを入れて",
          s"$RESET${GREEN}escキーを押してください",
          s"$RESET${DARK_GRAY}たまにアイテムが消失しますが",
          s"$RESET${DARK_GRAY}補償はしていません(ごめんなさい)",
          s"$RESET${DARK_GRAY}神に祈りながら交換しよう",
          s"$RESET$DARK_RED${UNDERLINE}クリックで開く"
        )
```

nits: `lore`は可変長引数も受け付けるため、こちらでも良いと思われます

Reply:
ありがとうございます、おそらくhttps://github.com/GiganticMinecraft/SeichiAssist/pull/1164/commits/63434ef294ba1e4719a8ff5a628a49d908013291 にて変更しました。(コミットを大きくしてしまい、わかりにくくなっていると思います。申し訳ございません...)